lib/main/widgets/pet_list_item.dart

```diff
@@ -0,0 +1,117 @@
+                color: Colors.white),
+            width: 221.0,
+            height: 92.0,
+            child: Padding(
+              padding:
+                  const EdgeInsets.symmetric(vertical: 8.0, horizontal: 9.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Row(
+                        children: [
+                          IconButton(
+                            icon: const Icon(
+                              Icons.folder_shared_outlined,
+                              size: 28.0,
+                              color: Colors.black,
+                            ),
+                            onPressed: () {},
+                          ),
+                          Text(
+                            namePet,
+                            style: GoogleFonts.montserrat(
+                              fontSize: 25.0,
+                              fontWeight: FontWeight.w500,
+                              color: Colors.black,
+                            ),
+                          )
+                        ],
+                      ),
+                      Row(
+                        children: [
+                          IconButton(
+                            icon: const Icon(
+                              Icons.pets,
+                              size: 28.0,
+                              color: Colors.black,
+                            ),
+                            onPressed: () {},
+                          ),
+                          Text(
+                            breedPet,
+                            style: GoogleFonts.montserrat(
+                              fontSize: 15.0,
+                              fontWeight: FontWeight.w600,
+                              color: const Color.fromRGBO(0, 0, 0, 0.7),
+                            ),
+                          )
+                        ],
+                      ),
+                    ],
+                  ),
+                  Column(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      IconButton(
+                        icon: const Icon(
+                          Icons.edit,
+                          size: 28.0,
+                          color: Colors.black,
+                        ),
+                        onPressed: () {},
+                      ),
+                      IconButton(
+                        icon: const Icon(
+                          Icons.restore_from_trash_outlined,
+                          size: 28.0,
+                          color: Colors.black,
+                        ),
+                        onPressed: () {},
+                      ),
```

Comment:
I think we should add a VoidCallback here for onPressed. (onPressed: someVoidCallback)